core/controllers/classifier.py

```diff
@@ -123,6 +125,24 @@ def post(self):
             classifier_services.get_classifier_training_job_by_id(job_id))
         if classifier_training_job.status == (
                 feconf.TRAINING_JOB_STATUS_FAILED):
+            # Send email to admin and admin specified email recipients.
+            # Other email recipients are specified on admin config page.
+            mail_subject = 'Failed ML Job %s' % job_id
+            admin_mail_body = ((
+                'ML job %s has failed. For more information,'
+                'please visit the admin page.') % job_id)
```

Comment:
Maybe give a direct link and clear instructions on what exactly to do?

As long as there are clear instructions to get to the failing job, it's fine.